ts/StudioEditor.ts

```
@@ -1,7 +1,7 @@
 import {GvasString, GvasText, gvasToString} from './Gvas';
 import {IndustryName, IndustryType, industryNames, isIndustryName, legacyIndustryNames} from './industries';
 import {Permission, permissionEqual, permissionLabels, permissionToString} from './Permission';
-import {fp32q, fp32r, fp32v, stringToText, textToString} from './util';
+import {toFloatingPointQuat, toFloatingPointRot, toFloatingPointVec, stringToText, textToString} from './util';
```

ESLint

[error] Member 'stringToText' of the import declaration should be sorted alphabetically.

(sort-imports)